Restoring Backup with an Existing Seed.md

```diff
@@ -0,0 +1,78 @@
+# Restoring Backup with an Existing Seed
+
+If you have already owned a Skywallet to store your coins and have bought a new one to replace it, you can opt to restore the old Skywallet using the new one.
+
+In order to restore your coins, you would only need the Seed of the old Skywallet.
```

Comment:
`coins, you only need Seed of another Skywallet.`

Reply:
Add a note here, that you can also use the seed of a desktop wallet, but this could compromise the security of the device.